### PR TITLE
maximum of 20 dollars per job

### PR DESCRIPTION
**Pull request by RobinL**

### CODE REVIEW

File: etl_manager/etl.py [hunk 1]
```diff
@@ -112,16 +113,30 @@ def __init__(
...
+
         self._glue_version = "1.0"
         self._python_version = "3"
 
+
+    @property
+    def timeout(self):
+        if self.timeout_override_minutes is None:
+            # 20 dollar limit, at 44 cents per worker per hour
+            return int(60 * (20 /(0.44 * self.allocated_capacity)))
```
Comment (samtazzyman) [inline etl_manager/etl.py]: Shouldn't we move these numbers out to be parameters, in case we want to move the goalposts to $10 or $40 later?

Comment (RobinL) [reply etl_manager/etl.py]: You can override these default values with

```
   g = GlueJob(
            "example/glue_jobs/simple_etl_job/",
            bucket="alpha-everyone",
            job_role="alpha_user_isichei",
            job_arguments={"--test_arg": "this is a test"},
            timeout_override_minutes = 2880
        )
```

So this is just providing a sensible failsafe for people who don't bother to think about cost....

I suppose we could set the `timeout_override_minutes` param to instead be `max job cost`.  That would make timeout number of minutes more obscure.

dunno...

Comment (RobinL) [reply etl_manager/etl.py]: See new commit....dont know if fully addresses but the chosen constants are at least more transparent now

**Comment by RobinL** [comment]
Glue sets a default timeout on all jobs of 2880 = 48h, so I just made the default that....

**Pull request merged**